### DISCUSSION TITLE
Configuration backup

### DISCUSSION THREAD

**Discussion by SpiderMANek** [AdguardTeam/AdGuardHome#7337] [Ideas]
Hi.
Please consider to adding a backup option to AdGuardHome.

**Comment by Dynamic5912** [comment]
Just take a copy of the YAML file

**Comment by SpiderMANek** [comment]
That is not saving added clients and all set options after deploy...

wt., 8 paź 2024, 21:32 użytkownik Dynamic5912 ***@***.***>
napisał:

**Comment by Dynamic5912** [comment]
> That os not saving added clients and all set options after deploy...
> 
> 
> 
> wt., 8 paź 2024, 21:32 użytkownik Dynamic5912 ***@***.***>
> 
> napisał:
> 
> 
> 
> > Just take a copy of the YAML file
> 
> >
> 
> > —
> 
> > Reply to this email directly, view it on GitHub
> 
> > <https://github.com/AdguardTeam/AdGuardHome/discussions/7337#discussioncomment-10884469>,
> 
> > or unsubscribe
> 
> > <https://github.com/notifications/unsubscribe-auth/BLKBTVVAVGGTNAV2MN2SGYDZ2QXNBAVCNFSM6AAAAABPSZ5HU2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOBYGQ2DMOI>
> 
> > .
> 
> > You are receiving this because you authored the thread.Message ID:
> 
> > ***@***.***
> 
> > com>
> 
> >
> 
> 

Yes it does...

**Reply by SpiderMANek** [reply]
Maybe it is simple for you, but I am new in dockers...
I want to change one of my drives on openmediavault server, on that drive I have installed many dockers, and AdGuard Home, I don't want to configure everything from zero after change of this drive, data folders are copying now woth rsync to new bigger one. After that i will change the drive and of cours change configuration of shared folder used by docker compose service on omv, the I will deploy portainer again, and rest of dockers.
One problematic is AdGuard Home, cause I've spend few hours to configure it for my purposes, I don't want to do it again.
Pleas write in steps what i should do to save my AdGuard Home stats and configuration?

**Reply by Dynamic5912** [reply]
As already advised. Locate and take a copy of the YAML file. Then if you ever need to restore AGH you just copy the YAML file over.

**Reply by SpiderMANek** [reply]
I know how to deploy stack and docker, but are you sure all data, and config will be used by new stack?
I hope it will, I will make backup of all dockers for security ;)
